JustSaying/AwsTools/MessageHandling/SqsNotificationListener.cs

```diff
@@ -98,21 +98,20 @@ public SqsNotificationListener WithMessageProcessingStrategy(IMessageProcessingS
 
         public void Listen(CancellationToken cancellationToken)
         {
-            var queue = _queue.QueueName;
+            var queueName = _queue.QueueName;
             var region = _queue.Region.SystemName;
-            var queueInfo = $"Queue: {queue}, Region: {region}";
 
             // Run task in background
             // ListenLoop will cancel gracefully, so no need to pass cancellation token to Task.Run
             _ = Task.Run(async () =>
             {
                 await ListenLoop(cancellationToken).ConfigureAwait(false);
                 IsListening = false;
-                _log.LogInformation($"Stopped Listening - {queueInfo}");
+                _log.LogInformation("Stopped listening on queue '{QueueName}' in region '{Region}'.", queueName, region);
```

Comment:
I _think_ that structured logging already puts quotes around strings in log messages, so the usages of `'` can all be removed.

Reply:
Can you verify before I remove them all?

Reply:
If I were to verify it it would be deleting the 0 in an IDE and seeing if it complies.